### PR TITLE
SPCTR-3566 Fix: Global - Resolved an issue wherein the template styles for frontend and single post template would not load as intended in FSE themes.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3566

### Screenshots

- https://d.pr/v/QPPWX2

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Global - Resolved an issue wherein the template styles for frontend and single post template would not load as intended in FSE themes.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Global - Resolved an issue wherein the template styles for the front-page and single-post templates would not load as intended in FSE themes.
```

**Review by bsf-zanev — COMMENTED** [review]
Just a changelog update required.

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Done

**Review by bsf-zanev — APPROVED** [review]

File: classes/class-uagb-front-assets.php [hunk 2]
```diff
@@ -120,7 +120,7 @@ public function enqueue_asset_files() {
 			if ( 0 !== $current_object_id && null !== $current_object_id ) {
 				$current_post_assets = new UAGB_Post_Assets( $current_object_id );
 				$current_post_assets->enqueue_scripts();
-			} elseif ( ! empty( $cached_wp_query ) && is_array( $cached_wp_query ) ) {
+			} elseif ( ! ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) && ! empty( $cached_wp_query ) && is_array( $cached_wp_query ) ) {
```
Comment (adi3890) [inline classes/class-uagb-front-assets.php:123]: lets keep this as a helper function as it is used in multiple places

**Review by adi3890 — APPROVED** [review]

File: classes/class-uagb-front-assets.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-front-assets.php:123]: We will be resolving above comment in this task - https://brainstormforce.atlassian.net/browse/SPCTR-3592

File: readme.txt [hunk 3]
```diff
@@ -217,6 +217,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 = x.x.x =
 * Fix: Global - Resolved an issue wherein the Astra customizer typography settings would not appear as expected.
 
+= x.x.x =
```
Comment (adi3890) [inline readme.txt:220]: lets merge two entry points in changelog

Comment (AbhijitNage123) [reply readme.txt]: Done

**Review by adi3890 — APPROVED** [review]

**Pull request merged**